Detectors/Upgrades/IT3/simulation/src/Detector.cxx

```diff
@@ -161,7 +161,10 @@ void Detector::configITS(Detector* its)
     its->defineInnerLayerITS3(idLayer, layerData[0], layerData[1], SensorLayerThickness, 0, 0);
     ++idLayer;
   }
-  its->createOuterBarrel(kTRUE);
+
+  // Redefine Wrapper Volume 0 using information on IB radii
+  static constexpr float safety = 0.5; // Educated guess, may be improved
+  its->defineWrapperVolume(0, tdr5data[0][0] - safety, tdr5data[3][0] + safety, wrpZSpan[0]);
```

Comment:
@mario6829 does not this repeat the volume for iw=0 in 
https://github.com/AliceO2Group/AliceO2/blob/6032242ff9cb660296b55dba3ee30143f87797a3/Detectors/Upgrades/IT3/simulation/src/Detector.cxx#L124-L126

Reply:
Hi @shahor02
as the comment says, it is done on purpose because only at this stage of the code the IB radii are known (they are defined at lines 149-153). I preferred not to fix the wrapper radii but compute them from innermost and outermost IB cylinders in order to leave the possibility of changing the IB radii - in case this is needed for further studies - without running the risk of extrusions or worrying about changing other lines.
But if you think it is cleaner, I can define only wrappers 1 and 2 in lines 124-125, or I can move altogether the definition of all wrappers in lines 166-167. Please let me know your suggestions.
Many thanks. Cheers
Mario

Reply:
@mario6829 OK, sorry, I did not realize that this is only to book the parameters and the actual volume additions follows later.